acoustid/data/submission.py

```
@@ -69,7 +69,9 @@ def import_submission(ingest_db, app_db, fingerprint_db, index_pool, submission)
     Import the given submission into the main fingerprint database
     """
 
-    fingerprint_hash = simhash(array.array("i", submission["fingerprint"]), signed=True)
+    fingerprint_hash = compute_simhash(
+        array.array("i", submission["fingerprint"]), signed=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)